libraries/lib-strings/UnusedStrings.h

```diff
@@ -78,3 +78,21 @@ XP("Got down one time", "Got down %d times", 0),
 XPC("Got down one time", "Got down %d times", 0, "pillows"),
 
 // //////////////////////////////////////////////// End examples
+
+
+// Crash & error report window
+
+XO("Problem Report for Audacity"),
+XO("Click “Send” to submit the report to Audacity. This information is collected anonymously."),
```

Comment:
I notice "smart" quotes automatically inserted by your editor.

This makes me think it would be good to apply them consistently.  To do that without changing many lines of the catalog, perhaps the program could make that substitution at runtime.

But as the catalog is now, "dumb" quotes would be consistent.

Reply:
And to put "dumb" quotes into this string so that it compiles, you must escape them with backslash:

\"